### ISSUE TITLE
No mention of not_contain_loss in paper?

### ISSUE THREAD

**Issue by srihari-humbarwadi** [yxlijun/tensorflow-yolov1#1]
This is term supposed to be there in the loss?
not_contain_loss = tf.nn.l2_loss(box_pred_response[:,4]-box_target_response[:,4])
@yxlijun
